makelib/common.mk

```diff
@@ -340,6 +340,10 @@ test:
 	@$(MAKE) test.run
 	@$(MAKE) test.done
 
+# integration tests
+test-integration:
+	@${ROOT_DIR}/cluster/local/integration_tests.sh
```

Comment:
sorry, I meant that we make `test-integration` a target in the Crossplane project, not here. Specifically I would consider the following:

```
test-integration: $(HELM) $(KIND) $(KUBECTL)
    @$(ROOT_DIR)/cluster/local/integration_tests.sh
```

and make the $(KIND) and $(KUBECTL) new targets possibly in `makelib\kubernetes.mk` similar to https://github.com/upbound/build/blob/master/makelib/kubebuilder.mk#L59

Reply:
+1 for putting the target which calls the integration test script in the Crossplane repo